.travis.yml

```diff
@@ -1,12 +1,18 @@
 language: go
-sudo: false
+
 go:
   - 1.7.x
   - 1.8.x
   - 1.9.x
   - tip
+
+sudo: false
+
+dist: trusty
```

Comment:
What's the benefit of explicitly declaring trusty? We have to manually update this if trusty becomes obsolete.

Reply:
I thought it would be needed in order to force the use of the Container-based build environment, but maybe not.

Reply:
When I added `sudo: false` I noticed speedup immediately and if I compare the duration of this PR's travis job and the one where `sudo: false` was intrudoced I cannot see significant differences:
 - https://travis-ci.org/gocolly/colly/builds/325491037
 - https://travis-ci.org/gocolly/colly/builds/325611501

Reply:
Then maybe I should remove that commit? (I also wonder why `"PATH=$GOPATH/bin:$PATH"` is set in the env section, `$GOPATH/bin` is automatically added to the $PATH by Travis-CI)

Reply:
You're right, then could you remove these lines?
If it's done, I'm merging this PR and migrating the new examples to the documentation site (I have a "beautiful" oneliner for this purpose =] ).